theorems/T000743.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: T000743
+if:
+  and:
+    - P000216: true
+    - P000164: true
+    - P000003: true
```

Comment:
```suggestion
    - P000003: true
    - P000164: true
```
this order seems more natural (keep T2 next to paracompact and put the non-measurable cardinal stuff at the end)